server/src/main/java/org/opensearch/action/termvectors/TransportTermVectorsAction.java

```diff
@@ -53,6 +54,8 @@
 
 import java.io.IOException;
 
+import static org.opensearch.action.get.TransportGetAction.shouldForcePrimaryRouting;
```

Comment:
Not sure if we should be getting this from TransportGetAction. See if there's a common place we can move this method to for use in both classes.

Reply:
I don't see any common place like parent classes that all 3 (TransportGet, TransportMultiGet, TransportTermVector) of these uses.

Reply:
Given the condition is pretty straightforward - it might be better to duplicate than import from another transport action.